cloud-controlplane/x-topics/quickstart.md

```diff
@@ -24,17 +24,27 @@ If you successfully retrieve an access token, it is valid for one hour. You can
 
 ### BYOC or Dedicated
 
-1. In the page header, click **API Explorer**.
+1. Open **API Explorer**.
```

Comment:
I think this quickstart needs to introduce the API Explorer a little better

Reply:
Yes, there are doc team conventions that haven't yet been adopted by the Engineering org as guidelines or styles in writing API descriptions or docs in general. It's a larger conversation to be had and will require effort to enforce. I think if we can encourage ENG to use our docs MCP server, that might help.